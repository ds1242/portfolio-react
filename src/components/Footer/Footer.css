#footer-icon {
    width: 1vw;
    height: 1vw;
}