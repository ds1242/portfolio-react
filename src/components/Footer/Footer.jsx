import React from 'react';
import './Footer.css';

function Footer() {
    return (
        <footer className="footer mt-auto " datatest-id="footer">
            <div>
                <h5 id="info-id">My Info:
                <a href="https://github.com/ds1242" target="_blank" rel='noreferrer noopener'><i className="bi bi-github"></i></a> 
                <a href="https://www.linkedin.com/in/david-shaw1242" target="_blank" rel='noreferrer noopener'><i className="bi bi-linkedin"></i></a> 
                </h5>
                <h5 datatest-id="date">Copyright &copy; {new Date().getFullYear()}</h5>
            </div>

        </footer>
    )
}

export default Footer;