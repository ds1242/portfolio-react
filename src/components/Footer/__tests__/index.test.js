import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from '..';

afterEach(cleanup);

describe('Footer component', () => {
    it('renders', () => {
        render(<Footer />);
    });

    it('matches snapshot DOM node structure', () => {
        const { asFragment } = render(<Footer />);

        expect(asFragment()).toMatchSnapshot();
    });
});

