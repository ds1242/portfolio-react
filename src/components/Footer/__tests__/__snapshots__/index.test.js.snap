// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <footer
    class="footer mt-auto "
    datatest-id="footer"
  >
    <div>
      <h5
        id="info-id"
      >
        My Info:
        <a
          href="https://github.com/ds1242"
          rel="noreferrer noopener"
          target="_blank"
        >
          <i
            class="bi bi-github"
          />
        </a>
        <a
          href="https://www.linkedin.com/in/david-shaw1242"
          rel="noreferrer noopener"
          target="_blank"
        >
          <i
            class="bi bi-linkedin"
          />
        </a>
      </h5>
      <h5
        datatest-id="date"
      >
        Copyright Â© 2021
      </h5>
    </div>
  </footer>
</DocumentFragment>
`;
